app/views/partials/GitRepositoryTeamNames.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cataloguefrontend.connector.GitRepository
+@import uk.gov.hmrc.cataloguefrontend.connector.model.TeamName
+
+@this()
+
+@(
+        repository: GitRepository
+)
+
+@if(!repository.isShared) {
+    @repository.teamNames.map {tn => <div  class="repo-team" ><a href="@uk.gov.hmrc.cataloguefrontend.teams.routes.TeamsController.team(TeamName(tn))">@tn</a></div> }
+} else {
+    <div title="@repository.teamNames.take(15).mkString("\n")" class="repo-team" >
+        <em><a href="@uk.gov.hmrc.cataloguefrontend.routes.CatalogueController.repository(repository.name)#teams">Shared by @repository.teamNames.length teams</a></em>
```

Comment:
I think we can leave this link out. It might be unexpected to the client to be taken to a different page when clicking on `Shared by ...` - it stands out more when not a link. There's usually a link already available on the repository name in the same table.

Reply:
The link is deliberate to make the table accessible and consistent.  For example, if someone uses a reader that zooms into part of the screen where the teams column is visible it's not ideal that they then have to scroll sideways to get to the link.